libs/build_miniconda3.sh

```diff
@@ -8,6 +8,8 @@ set -eux
 name="miniconda3"
 version=${2:-${STACK_miniconda3_version:-"latest"}}
 pyversion=${3:-${STACK_miniconda3_pyversion:-}}
+pyvenv=${4:-${STACK_miniconda3_pyvenv:-}}
```

Comment:
The job of this script `build_miniconda3.sh` is to build and install miniconda3 and not a conda env.
To install a conda env., one would install miniconda3 (this script), and then use the `libs/build_condaenv.sh` to build the conda env. using the miniconda3 installed by this script.

See examples in `develop` and instructions [here](https://hpc-stack.readthedocs.io/en/latest/hpc-notes.html#adding-a-new-library-or-package)

Reply:
Updated the recipe to build a regional_workflow conda environment, when it is requested in `./stack/stack...yaml` file. It uses an updated `./libs/build_condaenv.sh` script; no need to create an additional `build_...sh` script.